tools/cockpit.spec

```diff
@@ -444,12 +429,40 @@ fi
 %systemd_preun cockpit.socket cockpit.service
 
 %postun ws
-if [ -x %{_sbindir}/selinuxenabled ]; then
-    %selinux_modules_uninstall -s %{selinuxtype} %{name}
-    %selinux_relabel_post -s %{selinuxtype}
-fi
 %systemd_postun_with_restart cockpit.socket cockpit.service
 
+%package selinux-policies
+Summary: selinux policies required by cockpit
+# older -ws contained the SELinux policy, now split out
+Conflicts: %{name}-ws < 337-1.2025
+Requires(post): selinux-policy-%{selinuxtype} >= %{selinux_policyver}
```

Comment:
This used to be `>= %{_selinux_policy_version}`. Need to check if that works as well in Fedora/RHEL, otherwise we'll need a suse special case.

Reply:
`%selinux_policyver` doesn't exist in either [Fedora](https://src.fedoraproject.org/rpms/selinux-policy/blob/rawhide/f/rpm.macros) or [openSUSE](https://build.opensuse.org/projects/openSUSE:Factory/packages/selinux-policy/files/macros.selinux-policy).

Reply:
Done.